src/main/java/kata/makers/service/BookingService.java

```diff
@@ -0,0 +1,42 @@
+package kata.makers.service;
+
+import java.util.Optional;
+import java.util.Scanner;
+import kata.makers.enums.Role;
+import kata.makers.exception.CreateFlightReservationException;
+import kata.makers.model.Flight;
+import kata.makers.model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BookingService {
+
+  FlightService flightService;
+  FinancialService financialService;
+
+  private static final Logger log = LoggerFactory.getLogger(BookingService.class);
+
+  User merchant = new User(Role.MERCHANT, "Travel agency", 0.0);
+  User user = new User(Role.CUSTOMER, "jason", 1000.0);
```

Comment:
How would I go about creating a new user? Is that something to implement in the future?

Reply:
Good shout! I can have a try making it as a new feature